src/components/application/CmdInstructions.tsx

```diff
@@ -2,9 +2,14 @@ import { useTheme } from 'next-themes'
 import styles from './CmdInstructions.module.scss'
 import CodeCopy from './CodeCopy'
 
-const CmdInstructions = ({ appId }: { appId: string }) => {
+const CmdInstructions = ({
+  appId,
+  isStable,
```

Comment:
Can't this be an enum with Stable/Beta items, that represents the "Branch"? Who knows maybe at some point there would be a "nightly" thing or whatever

Reply:
It sure can, but changing that shouldn't be a problem at that point. This only designs for what we need right now, I fear scope creep otherwise.